proto/cosmos/app/v1alpha1/config.proto

```diff
@@ -4,6 +4,8 @@ package cosmos.app.v1alpha1;
 
 import "google/protobuf/any.proto";
 
+option go_package = "cosmossdk.io/api/app/v1alpha1";
```

Comment:
Added this option since https://github.com/grpc-ecosystem/grpc-gateway/issues/2127